code/game/objects/structures/wall_frame.dm

```diff
@@ -137,6 +136,9 @@
 /obj/structure/wall_frame/hull
 	color = COLOR_HULL
 
+/obj/structure/wall_frame/prepainted
```

Comment:
Not needed. The rest is a decent addition.

Reply:
...You already said this opinion above. People aren't so stupid to don't understand this

Reply:
The previous was a comment, this was me actually opposing it. There's a way GitHub works, you know.